daemon/cmd/daemon_main.go

```diff
@@ -1348,6 +1351,13 @@ func initEnv(vp *viper.Viper) {
 
 	switch option.Config.DatapathMode {
 	case datapathOption.DatapathModeVeth:
+	case datapathOption.DatapathModeNetkit, datapathOption.DatapathModeNetkitL2:
```

Comment:
nit / follow up for later : Should we probe for `netkit` support early ? Maybe even fallback to `veth` on missing support ?

Reply:
If a user configured netkit, I don't think we should fallback to veth, would be too confusing imho.